pyanaconda/bootloader.py

```diff
@@ -826,6 +826,12 @@ def set_boot_args(self, *args, **kwargs):
         if flags.cmdline.get("fips") == "1" and boot_device:
             self.boot_args.add("boot=%s" % self.stage2_device.fstabSpec)
 
+        #
+        # Spectre
```

Comment:
I would rename this to something like ``Mitigation to Spectre/Meltdown``

Reply:
Fixed.